app/scripts/controllers/network/network-controller.ts

```diff
@@ -379,6 +379,16 @@ function buildDefaultNetworkConfigurationsState(): NetworkConfigurations {
   return {};
 }
 
+function buildDefaultState() {
```

Comment:
Nit: Do we want JSDoc for this function?

Reply:
Oops, good catch! I meant to add this, I'll do that now

Reply:
Added here: https://github.com/MetaMask/metamask-extension/pull/18595/commits/9e42ffb44239fbaabc0e2f5f1779f0c3757c4973